src/tcia_utils/nbia.py

```diff
@@ -1021,14 +1041,15 @@ def viewSeries(seriesUid = "", path = ""):
 
     # error message function for when series doesn't exist or is invalid data
     def seriesInvalid(uid):
-        print("Cannot find a valid DICOM series at:")
-        print(path)
-        print("Try running downloadSeries(seriesUid, input_type = \"uid\") to download it first.")
-        print("If the data isn't restricted, you can alternatively view it in your browser (without downloading) using this link:")
-        if seriesUid != "":
-            print("https://nbia.cancerimagingarchive.net/viewer/?series=" + seriesUid)
+        if seriesUid:
+            link = f"https://nbia.cancerimagingarchive.net/viewer/?series={seriesUID}"
         else:
-            print("https://nbia.cancerimagingarchive.net/viewer/?series=YOUR_SERIES_UID")
+            link = "https://nbia.cancerimagingarchive.net/viewer/?series=YOUR_SERIES_UID"
+        _log.error(
+            f"Cannot find a valid DICOM series at: {path}\n"
+            'Try running downloadSeries(seriesUid, input_type = "uid") to download it first.\n'
+            f"If the data isn't restricted, you can alternatively view it in your browser (without downloading) using this link: {link}"
```

Comment:
I think we should print the link on a new line.  They are fairly long due to the length of the UIDs.

Reply:
Makes sense; it's been moved to a new line.